Extensions/CALayer+Extensions.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+import QuartzCore
+
+extension CALayer {
+  
+  public class func animateWithDuration(duration: TimeInterval, animation: () -> Void, completion: (() -> Void)?) {
+    CATransaction.begin()
+    CATransaction.setAnimationDuration(duration)
+    CATransaction.setCompletionBlock(completion)
+    animation()
+    CATransaction.commit()
+  }
+
+}
+
```

Comment:
extra line at end :)

Reply:
Why are we adding an extra line at the EOF?

Reply:
My guess is there might be 2 line at the end of file.

Reply:
Nope.. But I guess your file UIColor+extension dont have any extra line at the end  :D lol

Reply:
There is github just doesnot show one line :)